recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: bzip2
+  version: {{ version }}
+
+source:
+  fn: bzip2-{{ version }}.tar.gz
+  url: http://www.bzip.org/{{ version }}/bzip2-{{ version }}.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+  skip: true  # [win and py36]
+
+requirements:
+  build:
+    - toolchain
+    - python  # [win]
+
+test:
+  commands:
+    - bzip2 --help  # [unix]
```

Comment:
Can we please copy [these tests]( https://github.com/conda-forge/staged-recipes/pull/1977/files#diff-245f63c98c2671f2913ba880ae7d9a25R26 ) over too?